projects/core/src/cart/store/effects/cart.effect.spec.ts

```diff
@@ -34,9 +34,11 @@ const testCart: Cart = {
   },
 };
 
+const loadMock = createSpy().and.returnValue(of(testCart));
```

Comment:
Sharing one mock instance for all tests can lead to problems when calling i.e. `toHaveBeenCalled` and changing order of tests. Can we move the initialization of the mock and mockCartGenerator to the beforeEach() of test suite?